build.py

```diff
@@ -215,7 +218,7 @@ def compile_static_and_save(instance: ufoLib2.Font) -> None:
         optimizeCFF=ufo2ft.CFFOptimization.NONE,
     )
 
-    file_name = f"{family_name}-{style_name}".replace(" ", "")
+    file_name = name.replace(" ", "") + "-"  + instance.info.styleName
```

Comment:
I think this line has the same effect as it did before editing, but without the format interpolation it's a little less clear. any reason to change it? :)

Reply:
The family name is being set as “CascadiaMonoExtralight” so the filename becomes “CascadiaMonoExtralight-Extralight” whereas we wanted “CascadiaMono-Extralight”. That’s why I changed it.

Reply:
(As a side note, if we were build Mac platform naming, we could actually leave the family name ame as “Cascadia Mono” but that doesn’t work for the Microsoft platform. Instead we set the family name to “Cascadia Mono Light” (subfamily “Regular”) and then use the preferred family name fields to set “Cascadia Mono” (subfamily “Light”).

Reply:
Yeah.. wish it were that easy!

I'm still confused -- this code (skipping the bits in between) changes from..

```
file_name = f"{family_name}..."
```

to

```
name = family_name
file_name = name + "..."
```

I think they resolve to the same thing, but the concatenating version takes more mental effort to parse :)

Reply:
Huh. `f"{family_name}-{style_name}".replace(" ", "")` should stay because now `family_name = name` and `style_name = instance.info.styleName`, so this is a null-change. Also, the existing line despaces everything, style_name could potentially have a space.

Reply:
Oh I see. I think I may have confused myself. I’m ok with reverting that line.

Reply:
done